packages/storage/amplify_storage_s3/example/integration_test/issue_2711_test.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should we use a more descriptive key like getUrl_test / getUrl_special_characters / getUrl_signing ?

Reply:
Should add this test to `main_test.dart` to run as a part of the Storage integ test suite?